### PR TITLE
fix(profile): fix profile seed phrase backup badge

### LINKED ISSUE #4573: [base_bc] missing unread indicator (blue dot) for settings when seed phrase is not saved yet
### Description

Screenshot 2022-01-25 at 18 31 22

Gear icon should have a blue dot when the seed phrase was not yet saved by user

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #4573

### CODE REVIEW

File: src/app/modules/main/module.nim [hunk 1]
```diff
@@ -598,4 +602,14 @@ method resolvedENS*[T](self: Module[T], publicKey: string, address: string, uuid
...
+  self.view.activeSection().updateMember(publicKey, name, image, isIdenticon)
+
+method calculateProfileSectionHasNotification*[T](self: Module[T]): bool =
+  return not self.controller.isMnemonicBackedUp()
+
+method mnemonicBackedUp*[T](self: Module[T]) =
+  var profileItem = self.view.model().getItemById(conf.SETTINGS_SECTION_ID)
+  # I had to use this ugly syntax because `profileItem.hasNotification= ...` didn't compile
+  # This seems like a compilator error
+  `hasNotification=`(profileItem, self.calculateProfileSectionHasNotification())
```
Comment (saledjenic) [inline src/app/modules/main/module.nim:614]: @jrainville I agree it looks a bit ugly :) and not sure why this doesn't work, maybe cause `SectionItem` is not a `ref object`, not sure, but anyhow let's make it a bit nicer using methods we already have, if you don't mind just update this `mnemonicBackedUp ` method to this:
```
method mnemonicBackedUp*[T](self: Module[T]) =
  self.view.model().udpateNotifications(conf.SETTINGS_SECTION_ID, self.calculateProfileSectionHasNotification(), 0)
```

**Review by saledjenic — APPROVED** [review]
@jrainville looks good, I added just a minor improvement.

File: src/app/modules/main/module.nim (hunk 1 above)
Comment (jrainville) [reply src/app/modules/main/module.nim]: Good improvement. It worked and I updated

**Review by iurimatias — APPROVED** [review]
tested

**Pull request merged**